src/configure-balena-root-ca.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if test -n "${BALENA_ROOT_CA}"; then
+    echo "Installing self-signed balena root CA...";
+    echo "${BALENA_ROOT_CA}" | base64 -d > /usr/local/share/ca-certificates/balenaRootCA.crt;
+    update-ca-certificates;
+else
+    echo "No self-signed balena root CA found, using defaults...";
```

Comment:
Why the trailing semicolons?

Reply:
Habit, when I learned bash a long time ago, it's what you did. Now we only use them as separators on a line. Shall remove. 😅